apps/web/pages/user-edit/[id].tsx

```diff
@@ -0,0 +1,35 @@
+import UserEdit from '@/pages/UserEdit';
+import { addApolloState, initializeApollo } from 'apollo';
+import { getSession } from 'next-auth/react';
+import { GetServerSideProps } from 'next';
+
+import QUERY_GET_USER_FOR_PAGE from '../user/queryGetUserForPage.graphql';
+
+export default UserEdit;
+
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
```

Comment:
can you remove this function? no need to have SSR on this page.

Reply:
done